app/store/eventSlice.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable implicit-arrow-linebreak */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/prefer-default-export */
```

Comment:
Why?

Reply:
Taking theatest pull will resolve the issue